pyproject.toml

```diff
@@ -49,3 +49,5 @@ find={include = ["nir*"]}
 line-length = 100
 lint.per-file-ignores = {"docs/conf.py" = ["E402"]}
 exclude  = ["paper/"]
+extend-exclude = ["*.ipynb"]
```

Comment:
With version 0.6.0 of ruff, all notebooks are checked by default in ruff.
This lead to errors when checking the spyx example notebook: https://github.com/neuromorphs/NIR/actions/runs/10415679589.

To restore the previous behaviour, I excluded notebooks again.